integration/js/pages/TestAppPage.js

```diff
@@ -110,6 +133,153 @@ class TestAppPage {
     return false;
   }
 
+  async addVideoTileCheck(expectedTileId) {
+    let tileIdElement = await this.driver.findElement(elements.videoTileElementId);
+    if (!tileIdElement) {
+      return false;
+    }
+    let tileIdText =  await tileIdElement.getText();
+    if (!tileIdText) {
+      return false;
+    }
+    
+    if (expectedTileId === tileIdText) {
+      return true;
+    }
+    return false;
+  }
+
+  async getAllRemoteVideoTilesCheck(expectedTotalRemoteVideoTilesCount) {
+    let totalRemoteVideoTilesCountElement = await this.driver.findElement(elements.totalRemoteVideoTilesElementId);
+    if (!totalRemoteVideoTilesCountElement) {
+      return false;
+    }
+    let totalRemoteVideoTilesCount =  await totalRemoteVideoTilesCountElement.getText();
+    if (!totalRemoteVideoTilesCount) {
+      return false;
+    }
+    
+    if (expectedTotalRemoteVideoTilesCount === totalRemoteVideoTilesCount) {
+      return true;
+    }
+    return false;
+  }
+
+  async getAllVideoTilesCheck(expectedTotalVideoTilesCount) {
+    let totalVideoTilesCountElement = await this.driver.findElement(elements.totalVideoTilesElementId);
+    if (!totalVideoTilesCountElement) {
+      return false;
+    }
+    let totalVideoTilesCount =  await totalVideoTilesCountElement.getText();
+    if (!totalVideoTilesCount) {
+      return false;
+    }
+    
+    if (expectedTotalVideoTilesCount === totalVideoTilesCount) {
+      return true;
+    }
+    return false;
+  }
+
+  async getLocalVideoTileCheck(expectedLocalTileId) {
+    let localVideoTileIdElement = await this.driver.findElement(elements.localVideoTileElementId);
+    if (!localVideoTileIdElement) {
+      return false;
+    }
+    let localVideoTileId =  await localVideoTileIdElement.getText();
+    if (!localVideoTileId) {
+      return false;
+    }
+    
+    if (expectedLocalTileId === localVideoTileId) {
+      return true;
+    }
+    return false;
+  }
+
+  async hasStartedLocalVideoTileCheck(expectedHasStartedLocalVideoTileBoolean) {
+    let hasStartedLocalVideoTileBooleanElement = await this.driver.findElement(elements.hasStartedLocalVideoTileElementId);
+    if (!hasStartedLocalVideoTileBooleanElement) {
+      return false;
+    }
+    let hasStartedLocalVideoTileBoolean =  await hasStartedLocalVideoTileBooleanElement.getText();
+    
+    if (expectedHasStartedLocalVideoTileBoolean === hasStartedLocalVideoTileBoolean) {
+      return true;
+    }
+    return false;
```

Comment:
Seems like we could use a helper function to reduce repetitive code here

Reply:
Good point. Will change it.